build.gradle

```diff
@@ -14,6 +14,7 @@ repositories {
     maven { url 'http://repo.labs.intellij.net/central-proxy' }
     maven { url 'http://dl.bintray.com/jetbrains/intellij-plugin-service' }
     mavenCentral()
+    mavenLocal()
```

Comment:
I bet you set this repo after you run `gradle build` for this project, so all dependency were downloaded into gradle repo and mavenLocal won't be used anyway.